contracts/base/BaseAccount.sol

```diff
@@ -82,14 +82,13 @@ contract BaseAccount is IBaseAccount {
         address entryPointAddress = _ENTRYPOINT;
         /// @solidity memory-safe-assembly
         assembly {
-            mstore(0x14, to) // Store the `to` argument.
-            mstore(0x34, amount) // Store the `amount` argument.
-            mstore(0x00, 0x205c2878000000000000000000000000) // `withdrawTo(address,uint256)`.
+            mstore(0x14, to) // Store the `to` argument directly at memory location 0x14.
+            mstore(0x34, amount) // Store the `amount` argument at memory location 0x34.
+            mstore(0x00, 0x205c2878000000000000000000000000) // Store the function selector at memory location 0x00.
```

Comment:
Technically the selector is being stored at `0x10`

Reply:
It was failing so I just reapplied whatever was mentioned in the audit